### ISSUE TITLE
Reindexing project when IDE restart

### ISSUE THREAD

**Issue by Kontrael** [JetBrains/idea-gitignore#559]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [ ] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description
When the plugin is enabled. When you run the IDE, the project is fully indexed each time. This may take a few minutes.
If you disable the plugin, reindexing does not occur.

### Steps to Reproduce

1. To enable the plugin
2. To Restart The Storm
3. See the reindex

**Expected behavior:**
Reindexing should not occur

**Actual behavior:** 
There is re-indexing

**Reproduces how often:** 
Every time

### Versions

**Plugin:** 
```
3.0.0.182
```

**IDE:** 
```
PhpStorm 2018.2.1
Build #PS-182.3911.43, built on August 8, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

**OS:** 
```
Windows 10 10.0
```

**Comment by alshain** [comment]
I'm experiencing the same thing, also on Windows 10, but I'm running InteliJ EAP 2018.2.X.

**Comment by PhenX** [comment]
I'm experiencing the same thing, project opening is way faster after disabling the plugin.

> JetBrains Rider 2019.2 EAP
> Build #RD-192.5308.630, built on July 5, 2019
> Rider EAP User
> Expiration date: August 4, 2019
> Runtime version: 11.0.3+12-b304.10 amd64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
> Windows 10 10.0
> GC: G1 Young Generation, G1 Old Generation
> Memory: 2048M
> Cores: 4
> Registry: ide.tree.horizontal.default.autoscrolling=false, ide.tooltip.initialDelay.highlighter=0, search.everywhere.settings=true, parameter.info.max.visible.rows=10, ide.win.file.chooser.native=true, search.everywhere.pattern.checking=false, ide.tooltip.initialDelay=0, vcs.annotations.preload=true, ide.require.transaction.for.model.changes=false
> Non-Bundled Plugins: CMD Support, com.alayouni.ansiHighlight, com.intellij.ideolog, com.jetbrains.annotations.preloader, com.ppolivka.plugin.gitcleaner, in.thekalinga.snippet.intellij, io.wisetime.plugins.window.branch, myBitbucket, net.seesharpsoft.intellij.plugins.csv, net.vektah.codeglance, intellij-shellscript, izhangzhihao.rainbow.brackets, com.intellij.plugin.adernov.powershell, ua.in.dej.myEmmet, uk.co.ben-gibson.remote.repository.mapper

**Comment by dpinol** [comment]
I have this issue also with webstorm 2019.2 but I don't find the gitignore plugin.

**Comment by hsz** [comment]
Fixed with `v4.0.0`.
